src/less/success-message/base/success-message.less

```diff
@@ -40,6 +20,25 @@
     padding: 0 9%;
     text-align: center;
     width: 100%;
+
+    .btn--secondary,
+    .fake-btn--secondary {
+        border-color: @success-message-button-border-color;
+        color: @success-message-button-color;
+        cursor: pointer;
+        margin-top: 28px;
+
+        &:hover {
+            background-color: @success-message-button-color-hover;
+            border-color: @success-message-button-color-hover;
+            color: @success-message-button-color;
+            text-decoration: none;
```

Comment:
Do we need `text-decoration: none`? I thought button already had no `text-decoration`.

Reply:
Yep, I didn't catch that.  I'll remove.